Sources/Core/DTOLoader.swift

```diff
@@ -0,0 +1,25 @@
+//  DTOLoader.swift
+
+import Foundation
+import Yams
+
+struct DTOLoader {
+
+    enum GeneratorError: Error {
+        case invalidFormat(String)
+    }
+
+    func loadDto<T: Decodable>(url: URL) throws -> T {
+        let data = try Data(contentsOf: url)
+        switch url.pathExtension {
+        case "json":
```

Comment:
Maybe we should deprecate JSON specs in the future

Reply:
Are you saying JSON is the new XML?

_pretends to be shocked_